src/Microsoft.ML.StaticPipe/LdaStaticExtensions.cs

```diff
@@ -20,8 +20,8 @@ public sealed class LatentDirichletAllocationFitResult
         /// <param name="result"></param>
         public delegate void OnFit(LatentDirichletAllocationFitResult result);
 
-        public LatentDirichletAllocationTransformer.LdaSummary LdaTopicSummary;
-        public LatentDirichletAllocationFitResult(LatentDirichletAllocationTransformer.LdaSummary ldaTopicSummary)
+        public LatentDirichletAllocationTransformer.LdaModelParameters LdaTopicSummary;
```

Comment:
>LdaModelParameters [](start = 52, length = 18)

Since the model paramaters is an internal class, why not call it ModelParameters instead of LdaModelParameters?